src/app/pages/HomePage/HomePage.tsx

```diff
@@ -118,8 +120,9 @@ const HomePage = ({ pageData }: HomePageProps) => {
                       nthCurationByStyleAndProminence={
                         nthCurationByStyleAndProminence
                       }
+                      embed={embed}
                     />
-                    {index === 0 && <MPU />}
+                    {index === 1 && <MPU />}
```

Comment:
Although this change has been reverted, I would like to have do a show & tell with Product & UX with ads enabled, so that we see what this actually looks like